scripts/bash/run_enceladus.sh

```diff
@@ -208,11 +225,20 @@ validate() {
     fi
 }
 
+validate_either() {
```

Comment:
This is a bit weird. Because usually, 1st is the name of parameter and then the value. What lead you to switch the order?

Reply:
Yeah, it makes sense to revert the order in `valudate()` and `validate_either()`.